src/algorithms.jl

```diff
@@ -20,12 +20,6 @@ abstract type OrdinaryDiffEqExtrapolationVarOrderVarStepAlgorithm <: OrdinaryDif
 struct FunctionMap{scale_by_time} <: OrdinaryDiffEqAlgorithm end
 FunctionMap(;scale_by_time=false) = FunctionMap{scale_by_time}()
 
-@enum Controller begin
```

Comment:
why not use enums?

Reply:
It will be harder for the users to change the controllers, and plus, it will be a breaking change.